src/client/pages/CountryHome/Overview/hooks/index.ts

```diff
@@ -1,16 +1,63 @@
 import { useMemo } from 'react'
 
+import { Areas } from 'meta/area'
+import { AssessmentName, AssessmentNames, Cycle, CycleName } from 'meta/assessment'
 import { DashboardItem } from 'meta/dashboard'
 
-import { useCycleRouteParams } from 'client/hooks/useRouteParams'
+import { useCycle } from 'client/store/assessment'
+import { useCountryIso } from 'client/hooks'
+import { useAssessmentRouteParams } from 'client/hooks/useRouteParams'
 
-import { useDashboardMeta } from './useDashboardMeta'
+import { forestArea } from '../meta/forestArea'
+import { forestAreaPercentOfLandArea } from '../meta/forestAreaPercentOfLandArea'
+import { forestAreaWithinProtectedAreas } from '../meta/forestAreaWithinProtectedAreas'
+import { forestGrowingStockAndCarbonDashboard } from '../meta/forestGrowingStockAndCarbon'
+import { forestOwnership } from '../meta/forestOwnership'
+import { naturallyRegeneratingForestArea } from '../meta/naturallyRegeneratingForestArea'
+import { primaryDesignatedManagementObjectiveDashboard } from '../meta/primaryDesignatedManagementObjective'
+import { primaryForestPercentOfForestArea } from '../meta/primaryForestPercentOfForestArea'
 
-export const useItems = (): Array<DashboardItem> => {
-  const { assessmentName, cycleName } = useCycleRouteParams()
-  const dashboardMeta = useDashboardMeta()
+type DashboardItemFactory = (cycle: Cycle, region: boolean) => DashboardItem
 
-  return useMemo<Array<DashboardItem>>(() => {
-    return dashboardMeta[assessmentName][cycleName]
-  }, [assessmentName, cycleName, dashboardMeta])
+const defaultDashboardItemFactories: Array<DashboardItemFactory> = [
+  forestArea,
+  forestGrowingStockAndCarbonDashboard,
+  forestAreaPercentOfLandArea,
+  primaryForestPercentOfForestArea,
+  forestAreaWithinProtectedAreas,
+  forestOwnership,
+  primaryDesignatedManagementObjectiveDashboard,
+  naturallyRegeneratingForestArea,
+]
+
+const dashboardItemFactoriesMap: Record<AssessmentName, Record<CycleName, Array<DashboardItemFactory>>> = {
+  [AssessmentNames.fra]: {
+    '2020': defaultDashboardItemFactories,
+    '2025': defaultDashboardItemFactories,
+  },
+}
+
+const getDashboardItemFactories = (
+  assessmentName: AssessmentName,
+  cycleName: CycleName
+): Array<DashboardItemFactory> => {
+  return dashboardItemFactoriesMap[assessmentName]?.[cycleName] || []
```

Comment:
if for given assessment, cycle combination we don't find any dashboard items, return an empty array